test/unit/custom-events/custom-event-aggregator.test.js

```diff
@@ -33,36 +32,34 @@ tap.test('Custom Event Aggregator', (t) => {
     )
   })
 
-  t.afterEach(() => {
-    eventAggregator = null
+  t.afterEach((ctx) => {
+    ctx.nr.eventAggregator = null
   })
```

Comment:
Technically this is not necessary because each test will have a new scope. Thus, the `ctx.nr.eventAggregator` from test A will not be the same one in test B. But having the `afterEach` clear it is not harming anything. So this comment is informational, not blocking.